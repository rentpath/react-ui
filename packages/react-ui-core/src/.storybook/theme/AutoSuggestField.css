.AutoSuggestField {
  display: flex;
  margin: 75px;
}

.AutoSuggestField .Dropdown,
.AutoSuggestField .Field {
  margin: 0;
  width: 300px;
}

.AutoSuggestField .Button {
  height: 40px;
  position: relative;
  left: -70px;
  top: 5px;
}

.AutoSuggestField .Card {
  height: auto;
  width: auto;
  text-align: left;
}

.AutoSuggestField .List {
  padding: 0;
}

.AutoSuggestField .ListItem {
  padding: 5px 0;
}

.AutoSuggestField .Highlighter-Match {
  background: none;
}

.AutoSuggestField .Menu {
  padding-left: 5px;
}

.clear .Button {
  left: -40px;
}
