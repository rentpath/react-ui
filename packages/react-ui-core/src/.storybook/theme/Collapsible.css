.Collapsible_Items-visible {
  display: block;
}

.Collapsible_Items-hidden {
  display: none;
}

.Collapsible .Label {
  min-width: 200px;
}
