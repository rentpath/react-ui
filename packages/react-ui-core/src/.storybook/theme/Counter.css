.Counter {
  display: flex;
  white-space: nowrap;
  padding: 10px;
  align-items: center;
}

.Counter .Label {
  display: inline-block;
  white-space: nowrap;
  padding-right: 10px;
}

.Counter_Controls {
  display: inline-block;
  white-space: nowrap;
}

.Counter .Text {
  display: inline-block;
  text-align: center;
  padding: 0 20px;
}

.Counter_Button span {
  color: white;
  border-radius: 5px;
  padding: 5px 15px;
  display: inline-block;
  background: #0175D7;
  cursor: pointer;
}

.Counter_Button .Button {
  border-radius: 50%;
  width: 38px;
}

.Counter .Button {
  color: #ffffff;
  font-weight: bold;
}
