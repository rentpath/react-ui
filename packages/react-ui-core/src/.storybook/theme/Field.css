.Input,
.Select,
.Textarea {
  background-color: #ffffff;
  box-shadow: 0 0.5px 1px 0 rgba(0, 0, 0, 0.08);
  border: solid 1px #e9e9e9;
  border-radius: 4px;
  padding: 16px 10px;
  font-size: 13px;
}

.Field {
  margin: 16px 0;
  display: flex;
  max-width: 600px;
  color: #000;
  align-items: center;
}

.Field .Label {
  padding-right: 10px;
}
