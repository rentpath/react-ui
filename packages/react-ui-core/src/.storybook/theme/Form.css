.Form {
  min-width: 500px;
}

.Form .Field {
  color:#000;
}

.Form .Text {
  text-transform: uppercase;
  padding-bottom: 10px;
  margin-bottom: 10px;
}

.Form .Label {
  flex: 1;
}

.Form .Input,
.Form .Select,
.Form .Textarea {
  flex: 4;
}
