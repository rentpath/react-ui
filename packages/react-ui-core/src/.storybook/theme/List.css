.List {
  display: flex;
  padding: 20px;
}

.List-horizontal {
  flex-direction: row;
  justify-content: space-around;
}

.List-horizontal .ListItem {
  width: 100px;
}

.List-vertical {
  flex-direction: column;
}

.ListItem {
  cursor: pointer;
}

.ListItem-highlight {
  color: #AAA;
}
