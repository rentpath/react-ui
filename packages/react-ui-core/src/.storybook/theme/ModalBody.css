.ModalBody {
  min-width: 300px;
  padding: 20px;
  background: #FFF;
  flex-direction: column;
  border-radius: 5px;
  box-shadow: 1px 1px 5px #666;
  position: relative;
}
