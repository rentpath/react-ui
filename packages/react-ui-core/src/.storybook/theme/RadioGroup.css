.RadioGroup {
  border-radius: 5px;
  padding: .5rem 1rem;
}
