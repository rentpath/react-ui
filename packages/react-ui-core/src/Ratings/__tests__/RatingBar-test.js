import React from 'react'
import { shallow } from 'enzyme'
import theme from './mocks/theme'
import ThemedRatingBar from '../RatingBar'

const RatingBar = ThemedRatingBar.WrappedComponent

describe('RatingBar', () => {
  it('renders empty Ratings when rating is not provided', () => {
    const wrapper = shallow(<RatingBar maxScore={5} theme={theme} />)
    expect(wrapper.html()).toMatchSnapshot()
  })

  describe('ratings', () => {
    it('fills 1 of 5 rating items', () => {
      const wrapper = shallow(<RatingBar score={1} theme={theme} />)
      expect(wrapper.html()).toMatchSnapshot()
    })

    it('fills 3.5 of 5 rating items', () => {
      const wrapper = shallow(<RatingBar score={3.5} theme={theme} />)
      expect(wrapper.html()).toMatchSnapshot()
    })

    it('fills max when total is more than maxScore', () => {
      const wrapper = shallow(<RatingBar score={10} theme={theme} />)
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
