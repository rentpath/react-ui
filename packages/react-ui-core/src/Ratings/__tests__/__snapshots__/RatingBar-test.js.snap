// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingBar ratings fills 1 of 5 rating items 1`] = `"<div class=\\"RatingBar\\"><div class=\\"RatingBar_Background\\"><div class=\\"RatingBar_Icons\\" style=\\"width:20%\\"></div></div></div>"`;

exports[`RatingBar ratings fills 3.5 of 5 rating items 1`] = `"<div class=\\"RatingBar\\"><div class=\\"RatingBar_Background\\"><div class=\\"RatingBar_Icons\\" style=\\"width:70%\\"></div></div></div>"`;

exports[`RatingBar ratings fills max when total is more than maxScore 1`] = `"<div class=\\"RatingBar\\"><div class=\\"RatingBar_Background\\"><div class=\\"RatingBar_Icons\\" style=\\"width:100%\\"></div></div></div>"`;

exports[`RatingBar renders empty Ratings when rating is not provided 1`] = `"<div class=\\"RatingBar\\"><div class=\\"RatingBar_Background\\"></div></div>"`;
